README.md

```diff
@@ -147,6 +147,49 @@ matched), unless the `emailDomainMustHaveDot` option is disabled.
 Use `LinkType.EMAIL` for this, and see [test cases
 here](src/test/java/org/nibor/autolink/AutolinkEmailTest.java).
 
+### Hash tag extraction
+Extract Twitter/Facebook/Instagram Hash tags such as #photooftheday. Hash tags need to start with `#` and ends with a
+space. In general hash tags do not contain special characters other than `_`. Also hash tags do not start with a
```

Comment:
"ends with a space" -> it does not necessarily end with a space, it can also be at the end of the string, right?